### PR TITLE
implement Project section and about myself section

### PR DESCRIPTION
**Pull request by shahzaman900**
In this PR I implement:
- Projects Section
- About My Self Section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -156,4 +156,151 @@ ul {
       }
     }
   }
+
+  #works {
```
Comment (BregornOriginal) [inline style.css:183]: Good work implementing the works section

- [ ] As this is a requirement, this section should be implemented with `grid`. Grid is an interesting and really useful property to make layouts with CSS. Sometimes will be useful `flex` and other times `grid`, it's for this that you need to implement both of them in this project, to show that you know about how to use them 💯

File: index.html [hunk 2]
```diff
@@ -62,5 +62,223 @@ <h1>I'm Shah Zaman. Glad to see you!</h1>
...
+						</div>
+						<div class="tag">
+							<button class="small">CSS</button>
+						</div>
+						<div class="tag">
+							<button class="small">Javascript</button>
+						</div>
+					</div>
+          <div class="card_action">
+						<button class="secondary">See Project</button>
```
Comment (BregornOriginal) [inline index.html]: - [x] Kindly add the following functions to your button in the project. Active, Hover, and Pressed to match with the design provided

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @sheezu900  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (4)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR title and description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — APPROVED** [review]
Hi @sheezu900 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights

- [x] Good Commit History :green_circle:
- [x]  Professional Readme used :green_circle:
- [x]  No Linter errors :green_circle:
- [x] Professional PR and title :green_circle: 

## Optional suggestions

- [x] Please ensure to add Live demo section in the future,  Including a live demo in the README file demonstrates transparency and builds trust with users. It shows that the project is open and accessible, allowing users to evaluate its functionality without any barriers. This transparency can enhance the credibility of the project and increase user confidence in its quality and reliability. :smiley: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**